devtools-html/debugger.html
# [QuickOpen] Filter file search if a project directory root is set

Fixes Issue: #5760

### Summary of Changes

* Limits the possible result set when doing a source search when there's a project root set

### Test Plan

1.  Go to https://davidwalsh.name
2.  Set the project root to `/davidwalsh.name/wp-content/themes/punky/js`
3.  Search "m"
4.  You should only see files in that root

### To Do
- [x] Filter sources even when there is no query
- [x] Remove the directory root from the item subtitle
